src/assets/css/app.css

```
@@ -2651,9 +2654,10 @@ textarea.form-control-lg {
 }
 
 .form-select:focus {
-  border-color: #98ddb5;
+  border-color: #a8c7fa;
+  border: 1px solid var(--bs-light);
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)